index.html

```diff
@@ -386,6 +386,27 @@
           a representation of the signing date.</dd>
         </dl>
       </section>
+
+      <section>
+        <h2>Revocation Model</h2>
+        <p>A <a>Verifiable Claim</a> may provide revocation information
+        by referring to a revocation scheme by adding the following
+        <a>property</a> to the <a>Verifiable Claims Model</a>:</p>
```

Comment:
Suggest replacing all the above text with the following simplified wording to better match the existing model language:

<p>Revocation information for the claims in the <a>Verifiable Claims Model</a> may be provided by adding the following <a>property</a>:

since you already explain in the revocation property description that it must be a revocation scheme.